sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql

```diff
@@ -189,149 +138,235 @@ android_metrics AS (
-  sponsored_tiles_dismissal_count,
-  sponsored_tiles_disable_count,
+  COALESCE(sponsored_tiles_dismissal_count, 0) AS sponsored_tiles_dismissal_count,
+  COALESCE(sponsored_tiles_disable_count, 0) AS sponsored_tiles_disable_count,
   NULL AS sponsored_tiles_enabled_at_startup
 FROM
-  desktop_events_1
-FULL JOIN
-  desktop_events_2
+  (SELECT * FROM unified_metrics WHERE normalized_os NOT IN ("Android", "iOS")) desktop_unified
+LEFT JOIN
+  clicks_main
 USING
-  (submission_date, client_id)
-INNER JOIN
-  desktop_clients
+  (client_id, submission_date)
+LEFT JOIN
+  impressions_main
+USING
+  (client_id, submission_date)
+LEFT JOIN
+  desktop_activity_stream_events
+USING
+  (client_id, submission_date)
+-- add experiments data
+LEFT JOIN
+  (
+    SELECT
+      submission_date,
+      client_id,
+      experiments
+    FROM
+      `moz-fx-data-shared-prod.telemetry.clients_daily`
+    WHERE
+      submission_date = @submission_date
+  )
 USING
   (submission_date, client_id)
 UNION ALL
--- ios
+  --- iOS
 SELECT
   @submission_date AS submission_date,
-  "mobile" AS device,
-  "iOS" AS os,
+  device,
   client_id,
-  `mozfun.norm.browser_version_info`(browser_version_info) AS browser_version_info,
-  experiments,
+  browser_version_info,
   country,
   locale,
   normalized_channel,
-  normalized_os_version,
-  profile_age_in_days,
+  normalized_os,
+  is_new_profile,
   sample_id,
-  sponsored_tiles_click_count,
-  sponsored_tiles_impression_count,
+  experiments,
+  COALESCE(sponsored_tiles_click_count, 0) as sponsored_tiles_click_count,
+  COALESCE(sponsored_tiles_impression_count, 0) as sponsored_tiles_impression_count,
   NULL AS sponsored_tiles_dismissal_count,
-  NULL AS sponsored_tiles_disable_count,
+  COALESCE(sponsored_tiles_disable_count, 0) as sponsored_tiles_disable_count,
   NULL AS sponsored_tiles_enabled_at_startup
 FROM
-  ios_events
-INNER JOIN
-  ios_clients
+  (SELECT * FROM unified_metrics WHERE normalized_os = "iOS") ios_unified
+LEFT JOIN
+  ios_data
 USING
   (submission_date, client_id)
+-- add experiments data
 LEFT JOIN
   (
     SELECT
-      submission_date,
-      client_id,
-      days_since_created_profile AS profile_age_in_days
+      client_info.client_id AS client_id,
+      ARRAY_AGG(DATE(submission_timestamp) ORDER BY submission_timestamp DESC)[
+        OFFSET(0)
+      ] AS submission_date,
+      mozfun.map.mode_last(
+        ARRAY_CONCAT_AGG(
+          mozfun.glean.legacy_compatible_experiments(ping_info.experiments)
+          ORDER BY
+            submission_timestamp
+        )
+      ) AS experiments
     FROM
-      `moz-fx-data-shared-prod.firefox_ios.clients_last_seen_joined`
+      `mozdata.firefox_ios.events_unnested`
     WHERE
-      submission_date = @submission_date
-  ) profile_age_data
+      DATE(submission_timestamp) = @submission_date
+    GROUP BY
+      client_info.client_id
+  ) experiments_info
```

Comment:
This isn't an issue for this PR, but I wonder if this same logic is being done in other tables and we should just make `experiments_info` its own derived table

Reply:
Or just add it to [unified metrics](https://github.com/mozilla/bigquery-etl/issues/3095)!